deploy/README.md

```diff
@@ -6,20 +6,27 @@ This directory contains the Helm chart for deploying the ECK operator, and chart
 
 ## ECK Operator Helm Chart Usage
 
+Add the Elastic Helm repository.
+
+```sh
+helm repo add elastic https://helm.elastic.co
+helm repo update
+```
+
 Install the CRDs and deploy the operator with cluster-wide permissions to manage all namespaces.
 
 ```sh
-helm install elastic-operator eck-operator -n elastic-system --create-namespace 
+helm install elastic-operator elastic/eck-operator -n elastic-system --create-namespace 
```

Comment:
See my comment here: https://github.com/elastic/cloud-on-k8s/pull/6746#discussion_r1182520072:

> I think this is done on purpose when you want to install the operator using the Helm Chart in this directory, not from the elastic repository.

Reply:
Sorry, I missed it. I think we should use #6746 to fix this, and not modify this backport.

Reply:
> Sorry, I missed it. I think we should use #6746 to fix this, and not modify this backport.

SGTM, I added the `v2.8.0` label to https://github.com/elastic/cloud-on-k8s/pull/6746